db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/add_provider.rb

```diff
@@ -1,21 +1,17 @@
+gem "net-ssh", "=4.0.0.alpha4"
 require 'net/ssh'
 
 DEPLOYMENT_TYPES = {
-  :origin            => "ManageIQ::Providers::Openshift::ContainerManager",
-  :enterprise        => "ManageIQ::Providers::OpenshiftEnterprise::ContainerManager",
-  :atomic            => "ManageIQ::Providers::Atomic::ContainerManager",
-  :atomic_enterprise => "ManageIQ::Providers::AtomicEnterprise::ContainerManager"
+  :origin                 => "ManageIQ::Providers::Openshift::ContainerManager",
+  :"openshift-enterprise" => "ManageIQ::Providers::OpenshiftEnterprise::ContainerManager",
```

Comment:
I'm using the dash for consistency with ansible installer, it expects getting it with a dash, i can sub the underscore with a dash when creating the inventory file also.